test/mlir/conversion/onnx_to_torch/maxpool/symetricmaxpool.onnx.mlir

```diff
@@ -0,0 +1,15 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<1x3x8x8xf32>) -> tensor<1x3x3x3xf32> attributes {input_names = ["0"], output_names = ["3"]} {
+//CHECK: [[STRIDE:%.]] = torch.prim.ListConstruct %int1{{_*[0-9]*}}, %int1{{_*[0-9]*}} :
+//CHECK: [[DILATION:%.]] = torch.prim.ListConstruct %int1, %int1 :
+//CHECK: [[PAD:%.]] = torch.prim.ListConstruct %int0, %int0{{_*[0-9]*}} :
+//CHECK: [[BIAS:%.]] = torch.prim.ListConstruct %int2, %int2{{_*[0-9]*}} :
+//CHECK: torch.aten.max_pool2d %arg0, [[BIAS]], [[STRIDE]], [[PAD]], [[DILATION]], %false : !torch.vtensor<[1,3,8,8],f32>, !torch.list<int>, !torch.list<int>, !torch.list<int>, !torch.list<int>, !torch.bool -> !torch.vtensor<[1,3,7,7],f32>    
```

Comment:
One `CHECK` but three `onnx.MaxPoolSingleOut` ops.

Reply:
Liam,
We have removed the additonal onnx.MaxPoolSingleout here. They did not contribute much to testing here.